backend/tracim_backend/models/userconfig.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.types import JSON
+from sqlalchemy.types import Integer
+
+from tracim_backend.models.auth import User
+from tracim_backend.models.meta import DeclarativeBase
+
+
+class UserConfig(DeclarativeBase):
+    """
+    User Config definition.
+    This table stores parameters for Tracim clients (including the Tracim frontend)
+    """
+
+    __tablename__ = "user_configs"
+
+    user_id = Column(
+        Integer,
+        ForeignKey(
+            "users.user_id",
+            name="fk_userconfig_user_id_users",
```

Comment:
This shouldn't be needed as sqlalchemy uses the defined naming convention for those.